zwave_js_server/model/endpoint.py

```diff
@@ -324,7 +324,7 @@ async def async_set_raw_config_parameter_value(
 
         data = await self.async_send_command(
             "set_raw_config_parameter_value",
-            options={k: v for k, v in options.items() if v is not None},
+            **options,
```

Comment:
Is it ok to remove the filtering of options set to `None` or should we keep that filter before unpacking the options?

Reply:
I think it needs to be filtered first